classic/form/field/GeocoderComboBox.js

```diff
@@ -220,6 +220,16 @@ Ext.define('GeoExt.form.field.GeocoderComboBox', {
         });
     },
 
+    setMap: function(map) {
```

Comment:
No need to overwrite setMap here IMHO. Just do 

``` javascript
if (me.getMap()) {
    me.getMap().addLayer(me.locationLayer);
}
```
after `me.callParent(arguments);`